README.md

```diff
@@ -11,14 +11,14 @@ An Ember CLI addon which allows you to specify CSS inside of component pod direc
 
 ## Usage
 
-This addon allows you to specify a `styles.css` file inside of your component's pod folder (`app/my-component/styles.css`) (the component must also provide a `component.js` returning a valid instance of `Ember.Component`).
+This addon allows you to specify a style sheet inside of your component's pod folder. (the component must also provide a `component.js` returning a valid instance of `Ember.Component`)
```

Comment:
could you remove the `(the component must also provide a `component.js` returning a valid instance of `Ember.Component`)` Thats no longer true in the beta version!